### ISSUE TITLE
Change from gh-pages to master

### ISSUE THREAD

**Issue by jxxcarlson** [HubPress/hubpress.io#169]
I had things setup for gh-pages but now want to change back to master.  How do I do that?  My attempts so far have failed:-)  Both branches are setup, but in Admin console I can't change the git-branch  Changing the default branch on the github side doesn't seem to work either.

**Comment by jaredmorgs** [comment]
So what have you done so far to convert your site back to /master from
/gh-pages? Can you provide steps to reproduce?

**Comment by jxxcarlson** [comment]
I changed the default branch from gh-pages to master.

**Comment by jaredmorgs** [comment]
I'm not sure of your knowledge of GitHub, so I'll tell you as much as I
think you should need and if you already know it, please excuse me.

If I understand your config correctly, your /master branch would be out of
sync with the work you've done on your blog in gh-pages. So what you need
to do is migrate your gh-pages content over to the master branch.

You can do that through the GitHub website through a Pull Request (PR).

You compare your gh-pages branch with your master branch. You've done it
right if you see a lot of suggested changes to master in the Pull Request.

Once you've created the Pull Request and applied it to /master, you then
need to adjust the configuration as described in the README.adoc

If you've never done a pull request before, watch the linked video in the
README for instructions.

Hope it helps

Jared

**Comment by jxxcarlson** [comment]
I followed your advice and the work in the gh-pages branch is merged into master.  I had to do some work at the command line for this to work.  Now the situtation is as follows (1) http://jxxcarlson.github.io loads but looks terrible.  I think it is not recognizing css files and also liks are out of date: follwoing them gives error 404; (2) I cannot edit the site: if I try to connect to http://jxxcarlson.github.io/hubpress/ , I get an infinitely spinning wheel of death -- the login page (or anything for that matter), never loads.  My defalut branch is now set to master.  Perhaps Hubpress is not aware of all the changes??
